desc/compute/_qs.py

```diff
@@ -301,6 +422,47 @@ def _B_modes(params, transforms, profiles, data, **kwargs):
     return data
 
 
+@register_compute_fun(
+    name="Boozer transform modes norm",
+    label="",
+    units="~",
+    units_long="None",
+    description="Inner product norm for boozer modes basis.",
+    dim=1,
+    params=[],
+    transforms={"B": [[0, 0, 0]]},
+    profiles=[],
+    coordinates="rtz",
+    data=[],
+)
+def _boozer_modes_norm(params, transforms, profiles, data, **kwargs):
+    # norm is 1 if m=n=0, 2 if m=0 or n=0, 4 if m!=0 and n!=0
+    norm = 2 ** (3 - jnp.sum((transforms["B"].basis.modes == 0), axis=1))
+    data["Boozer transform modes norm"] = norm
+    return data
+
+
+@register_compute_fun(
+    name="Boozer transform matrix",
+    label="",
+    units="~",
+    units_long="None",
+    description="Matrix which performs boozer transformation.",
+    dim=1,
+    params=[],
+    transforms={"B": [[0, 0, 0]]},
+    profiles=[],
+    coordinates="rtz",
+    data=["rho", "theta_B", "zeta_B", "Boozer transform modes norm"],
+)
+def _boozer_transform_matrix(params, transforms, profiles, data, **kwargs):
+    nodes = jnp.array([data["rho"], data["theta_B"], data["zeta_B"]]).T
+    data["Boozer transform matrix"] = (
+        data["Boozer transform modes norm"] * transforms["B"].basis.evaluate(nodes)
```

Comment:
Sure but this is already used with the boozer nodes, and is only used in the boozer transform part. I include the norm multiplication here to avoid unnecessary computations as much as necessary

Reply:
I still think the `"Boozer transform modes norm"` does not need to be its own compute quantity and could be moved into this function, because this is the only placed they are ever used. And the normalizations are somewhat meaningless on their own.